tests/bemClassName.spec.js

```diff
@@ -1,5 +1,5 @@
-import React from 'react';
-import { shallow, mount } from 'enzyme';
+import { h } from 'preact';
+import { deep } from 'preact-render-spy';
```

Comment:
What about making an abstraction layer to cover both of them?
Looks like bemzyne isn't so dead, huh?